packages/vitest-example-test-suite/src/test1.spec.ts

```diff
@@ -0,0 +1,7 @@
+import { expect, test } from "vitest";
+import { add } from "./add";
+
+test('adds 1 + 2 to equal 3', async () => {
+    await new Promise((resolve) => setTimeout(resolve, 5000));
```

Comment:
I think we don't need to make tests too slow here and in other test files. There is no need to verify if tests are well balanced across parallel nodes. This is tested elsewhere.

Let's keep most of the tests fast. We use `vitest-example-test-suite` mostly locally to verify the Vitest integration works while working on changes in it. We would like to have a quick feedback loop when running `npm run test:vitest`.

```suggestion
    await new Promise((resolve) => setTimeout(resolve, 1500));
```